lib/views/help/how.html.erb

```diff
@@ -244,6 +248,21 @@
     get a copy of it. We also want to save taxpayers’ money by preventing
     duplicate requests.
   </p>
+  <h4 id="compassionate_grounds">
+  Removal of information on compassionate grounds
+  <a href="#compassionate_grounds">#</a>
+  </h4>
+  <p>
+    On rare occasions, in exceptional circumstances, an editorial decision may be
+    taken to remove requests, and/or responses, on compassionate grounds. Such
+    cases are treated sensitively on a case-by case basis.
+  </p>
+  <p>
+    Where requests relate to deceased individuals we have a presumption towards 
+    removing material on compassionate grounds, where the request is by a 
```

Comment:
I think the comma here should be removed.

Reply:
It may make more sense to remove the comma after 'grounds', and add one before 'individuals' instead. What do you think?

It's a very minor change either way, so this can easily be taken care of by a simple fixup.

Reply:
I think no comma at all is best.

Reply:
Fixup posted - fcf6d0fdd31f848c037a0a02066b68da21c35fda